cli/cliui/provisionerjob.go

```diff
@@ -74,7 +80,20 @@ func ProvisionerJob(ctx context.Context, wr io.Writer, opts ProvisionerJobOption
 	sw := &stageWriter{w: wr, verbose: opts.Verbose, silentLogs: opts.Silent}
 
 	printStage := func() {
-		sw.Start(currentStage)
+		out := currentStage
```

Comment:
nit: does it have to operate on vars declared outside of the scope of this function?

Reply:
That's what the other functions do as well; I don't love it, TBH.
I actually started a refactor to move this state out into a struct, but for now this seemed like the simpler approach.